app/report/residency_fr_taxes_fr/ReportResidencyFrTaxesFr.tsx

```diff
@@ -268,6 +305,37 @@ export const ReportResidencyFrTaxesFr: React.FunctionComponent<
             </div>
             <Page510 taxes={taxes} isPrintMode={isPrintMode} />
           </Section>
+          <Section title="Source of information">
```

Comment:
Every section could use a little more space top / bottom, do you mind adding a gap between each section?